libs/output-mapping/azure-pipelines.tests.yml

```diff
@@ -103,3 +103,18 @@ jobs:
         STORAGE_API_TOKEN: $(OUTPUT_MAPPING_NATIVE_TYPES__STORAGE_API_TOKEN_AWS)
         STORAGE_API_TOKEN_MASTER: $(OUTPUT_MAPPING__STORAGE_API_TOKEN_MASTER_AWS)
         SYNAPSE_STORAGE_API_TOKEN: $(OUTPUT_MAPPING__SYNAPSE_STORAGE_API_TOKEN)
+
+  - template: ../../azure-pipelines/jobs/run-tests.yml
+    parameters:
+      jobName: tests_php74_tableWriter_slice
```

Comment:
8

Reply:
jel jsem copy paste, je to tam vsude :) upravimto nekdy